### ISSUE TITLE
Installation crashes without error if DISABLE_FIREWALL=1 is not passed on LXC

### ISSUE THREAD

**Issue by Forceu** [mail-in-a-box/mailinabox#2140]
When trying to install MIB on an Ubuntu 18.04 container, the installation fails without error output after initilising of the random number generator:

```
Installing packages needed for setup...
Primary Hostname: xxxx
Public IP Address: xxxx
Public IPv6 Address: xxxx
Private IP Address: xxxx
Private IPv6 Address: xxxx
Mail-in-a-Box Version:  v57a

Installing add-apt-repository...
Updating system packages...
Installing system packages...

Current default time zone: 'Europe/Berlin'
Local time is now:      Sat Jul  2 17:45:05 CEST 2022.
Universal Time is now:  Sat Jul  2 15:45:05 UTC 2022.

Initializing system random number generator...
```

If the setup is called with the `DISABLE_FIREWALL=1` argument (`DISABLE_FIREWALL=1 setup/start.sh`) it runs without any problems.

I am opening this issue as no error is output - if the issue is not fixed, maybe it helps at least someone who is having the same problem.

**Comment by DustinReynoldsPE** [comment]
Thanks for opening this, it helped me get un-stuck when installing MIAB on a Ubuntu 22.04 LXC in Proxmox.

**Comment by johncsuti** [comment]
YES Thank you so much for opening this issue and outlining a solution!

**Comment by cweiland** [comment]
Wow best response ever -> next step find why it's happend

**Comment by cweiland** [comment]
It's on code : 
# ### Firewall

# Various virtualized environments like Docker and some VPSs don't provide #NODOC
# a kernel that supports iptables. To avoid error-like output in these cases, #NODOC
# we skip this if the user sets DISABLE_FIREWALL=1. #NODOC`
